src/file.ts

```diff
@@ -1235,9 +1237,9 @@ class File extends ServiceObject<File> {
                   rawResponseStream.pipe(pumpify.obj(throughStreams));
             }
 
-            rawResponseStream.on('end', onComplete).pipe(throughStream, {
-              end: false
-            });
+            rawResponseStream.on('error', onComplete)
```

Comment:
Is having 2 error listeners with the same callback intentional?

Reply:
The code above this actually re-assigns what `rawResponseStream` is. When we did that, it was to avoid duplicating the event handler assignments.

Reply:
Oh ok, sorry missed that part.